src/main/java/com/poortorich/chat/controller/ChatController.java

```diff
@@ -90,4 +92,27 @@ public ResponseEntity<BaseResponse> leaveChatroom(
                 response
         );
     }
+
+    @DeleteMapping("/leave")
+    public ResponseEntity<BaseResponse> leaveAllChatroom(
+            @AuthenticationPrincipal UserDetails userDetails,
+            @RequestBody ChatroomLeaveAllRequest chatroomLeaveAllRequest
+    ) {
+        ChatroomLeaveAllResponse response = chatFacade.leaveAllChatroom(
+                userDetails.getUsername(),
+                chatroomLeaveAllRequest);
+
+        for (Long chatroomId : chatroomLeaveAllRequest.getChatroomsToLeave()) {
+            ResponsePayload payload = realTimeFacade.createUserLeaveSystemMessage(
+                    userDetails.getUsername(),
+                    chatroomId);
+
+            messagingTemplate.convertAndSend(SubscribeEndpoint.CHATROOM_SUBSCRIBE_PREFIX + chatroomId, payload);
+        }
+
+        return DataResponse.toResponseEntity(
+                ChatResponse.CHATROOM_LEAVE_SUCCESS,
+                response
+        );
```

Comment:
이 부분은 한 줄로 작성해도 괜찮을 거 같네요!

```suggestion
        return DataResponse.toResponseEntity(ChatResponse.CHATROOM_LEAVE_SUCCESS, response);
```